src/presentation/wallet/log-in/index.tsx

```diff
@@ -85,10 +104,7 @@ const LogIn: React.FC = () => {
         <h2 className="text-grayLight text-lg font-medium">
           The ultimate gateway to access the Liquid Network
         </h2>
-        <LogInEnhancedForm dispatch={dispatch} history={history} />
-        {/* <Link className="text-primary block font-bold text-left" to={RESTORE_VAULT_ROUTE}>
```

Comment:
This Link should be added back at some point

Reply:
Yes this is for #116

Reply:
@louisinger

Reply:
An issue is enough to remember, no need to keep commented code IMO but I can revert if u prefer